torch/csrc/jit/passes/onnx/shape_type_inference.cpp

```diff
@@ -1374,13 +1392,88 @@ void ONNXShapeTypeInference(
 
 } // namespace
 
+std::pair<bool, bool> IsShapeReliable(Node* n) {
```

Comment:
can there be a better naming for this? it is returning both reliable and complete, and it is associated with the inputs of node n, rather than the shapes (of outputs) of n.

Reply:
Rename as `AreInputsReliableOrStatic`